src/components/Dialog/Dialog.ts

```diff
@@ -46,99 +55,85 @@ export type NotifyAndNotShowAgainButtonOption = {
 export type LoadingScreenOption = { message: string };
 
 // 汎用ダイアログを表示
-export const showAlertDialog = async (
-  options: CommonDialogOptions["alert"],
-) => {
+export const showAlertDialog = async (options: TextAlertDialogOptions) => {
   options.ok ??= "閉じる";
 
-  return new Promise((resolve: CommonDialogCallback) => {
-    setCommonDialogCallback(
-      Dialog.create({
-        title: options.title,
-        message: options.message,
-        ok: {
-          label: options.ok,
-          flat: true,
-          textColor: "display",
-        },
-      }),
-      resolve,
-    );
-  });
+  const { promise, resolve } = Promise.withResolvers<void>();
+  Dialog.create({
+    component: MessageDialog,
+    componentProps: {
+      type: options.type ?? "info",
+      title: options.title,
+      message: options.message,
+      ok: options.ok,
+    },
+  }).onOk(() => resolve());
+
+  await promise;
+
+  return "OK" as const;
 };
 
-/**
- * htmlフラグを`true`にする場合、外部からの汚染された文字列を`title`や`message`に含めてはいけません。
- * see https://quasar.dev/quasar-plugins/dialog#using-html
- */
-export const showConfirmDialog = async (
-  options: CommonDialogOptions["confirm"],
-) => {
+export const showConfirmDialog = async (options: TextConfirmDialogOptions) => {
   options.cancel ??= "キャンセル";
 
-  return new Promise((resolve: CommonDialogCallback) => {
-    setCommonDialogCallback(
-      Dialog.create({
-        title: options.title,
-        message: options.message,
-        persistent: true, // ダイアログ外側押下時・Esc押下時にユーザが設定ができたと思い込むことを防止する
-        focus: "ok",
-        html: options.html,
-        ok: {
-          flat: true,
-          label: options.actionName,
-          textColor: "display",
-        },
-        cancel: {
-          flat: true,
-          label: options.cancel,
-          textColor: "display",
-        },
-      }),
-      resolve,
-    );
-  });
+  const { promise, resolve } = Promise.withResolvers<number>();
+  Dialog.create({
+    component: QuestionDialog,
+    componentProps: {
+      type: options.type ?? "question",
+      title: options.title,
+      message: options.message,
+      buttons: [options.cancel, options.actionName],
+      default: 0,
```

Comment:
以前の見る感じ、confirmは`ok`がデフォルトっぽみ？
```suggestion
      default: 1,
```